BEIMA.Backend/SampleFileStore.cs

```diff
@@ -8,30 +8,44 @@
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 using Azure.Storage.Blobs;
+using BEIMA.Backend.StorageService;
+using System.Collections.Generic;
 
 namespace BEIMA.Backend
 {
-    public static class SampleFileStore
+    public class SampleFileStore
```

Comment:
Is this something that we are using as a part of the actual product? If so we should have unit tests for it.

Reply:
Its kind of a remnant of the sample functions we have, though I just used it for manual testing. The others are SampleDB, SampleFunction, Function1. I'm somewhat tempted to remove them though as they don't contribute to the actual product.

Reply:
I think it would be a good idea to remove the other ones as they are probably just clutter at this point.

Reply:
I'll put that down as a new task